From 51d96ae13dd18372d875623edfe15339a2d58727 Mon Sep 17 00:00:00 2001
From: Kateryna Pietukhova <kateryna.pietukhova@sirisoftware.com>
Date: Mon, 17 Aug 2015 21:02:05 +0300
Subject: [PATCH 1/1] Build type option was added

---
 Makefile |    6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Makefile b/Makefile
index 91b6946..b33144d 100644
--- a/Makefile
+++ b/Makefile
@@ -19,6 +19,12 @@ world:
 
 include $(TOPDIR)/include/host.mk
 
+ifeq ($(BUILD_TYPE),DEBUG)
+  $(shell cp $(TOPDIR)/files/etc/init.d/sabaifs_debug $(TOPDIR)/files/etc/init.d/sabaifs)
+else
+  $(shell cp $(TOPDIR)/files/etc/init.d/sabaifs_release $(TOPDIR)/files/etc/init.d/sabaifs)
+endif
+
 ifneq ($(OPENWRT_BUILD),1)
   _SINGLE=export MAKEFLAGS=$(space);
 
-- 
1.7.9.5

