--- a/root/etc/config/openvpn_recipes
+++ b/root/etc/config/openvpn_recipes
@@ -11,6 +11,7 @@ config openvpn_recipe server_tun_ptp
 	option comp_lzo			"yes"
 	option verb			"3"
         option mssfix			"1420"
+	option explicit_exit_notify     "2"
 
 #
 # Routed point-to-point client
@@ -25,6 +26,7 @@ config openvpn_recipe client_tun_ptp
 	option nobind			"1"
 	option comp_lzo			"yes"
 	option verb			"3"
+	option explicit_exit_notify     "2"
 
 #
 # Routed multi-client server
@@ -42,6 +44,7 @@ config openvpn_recipe server_tun
 	option comp_lzo			"yes"
 	option verb			"3"
         option mssfix			"1420"
+	option explicit_exit_notify     "2"
 
 #
 # Routed client
@@ -61,6 +64,7 @@ config openvpn_recipe client_tun
 	option verb			"3"
 	option reneg_sec		"0"
 	option float			"1"
+	option explicit_exit_notify     "2"
 
 #
 # Multi-client ethernet bridge server
@@ -78,6 +82,7 @@ config openvpn_recipe server_tap_bridge
 	option comp_lzo			"yes"
 	option verb			"3"
         option mssfix			"1420"
+	option explicit_exit_notify     "2"
 
 #
 # Ethernet bridge client
@@ -99,4 +104,5 @@ config openvpn_recipe client_tap_bridge
 	option verb			"3"
 	option reneg_sec		"0"
 	option float			"1"
+	option explicit_exit_notify	"2"
 
